dev/ledger.md

```diff
@@ -107,6 +107,11 @@ The block header contains the following components:
    issued prior to this block.  This count starts from the first block with a
    protocol version that supported the transaction counter.  The counter is
    stored in msgpack field `tc`.
+
+ - The block's _participation updates_, which is the collection of account
+   data that represents information that needs to be acted upon due to changes
+   related to participation.  The msgpack representation of the components are
+   described in detail below.
```

Comment:
- Using the term "account data" here is ambiguous, since it doesn't contain any of the account data. 
- The _participation updates_ is not part of the protocol ( i.e. it's transparent.. ). It's the slice that is visible.

Instead, I would suggest phrasing it as -

The block's _expired participation accounts_, which contains optional slice of
accounts public key. These accounts are expected to have their participation 
key expire by the end of the round. The msgpack representation of the components are
described in detail below. The slice is stored in msg pack field `partupdrmv`.